### PR TITLE
Added option to customize caption location

### PR DESCRIPTION
**Pull request by smaltby**
Partially addresses issue #171. Currently only has English translations.

### CODE REVIEW

File: _locales/da/messages.json [hunk 1]
```diff
@@ -87,6 +87,26 @@
     "message":"Forøg størrelsen af forstørrede billeder med denne mændge",
     "description":"[options] Tooltip for adjust the size of zoomed images option"
   },
+  "optCaptionLocation": {
```
Comment (extesy) [inline _locales/da/messages.json]: please remove messages from all locales except for "en" - it's used as a default fallback locale

File: js/common.js [hunk 2]
```diff
@@ -34,6 +33,7 @@ function loadOptions() {
     options.enableGalleries = options.hasOwnProperty('enableGalleries') ? options.enableGalleries : true;
     options.ambilightEnabled = options.hasOwnProperty('ambilightEnabled') ? options.ambilightEnabled : false;
     options.disabledPlugins = options.hasOwnProperty('disabledPlugins') ? options.disabledPlugins : [];
+    options.captionLocation = options.hasOwnProperty('captionLocation') ? options.captionLocation : 'below';
```
Comment (extesy) [inline js/common.js]: This is not backwards compatible. All people who had disabled captions would suddenly be opted-in to these captions again. Please change `below` to either `below` or `none` depending on the existing `showCaptions` value.

File: js/hoverzoom.js [hunk 3]
```diff
@@ -593,8 +593,12 @@ var hoverZoom = {
 
             if (hz.currentLink) {
                 var linkData = hz.currentLink.data();
-                if (options.showCaptions && !options.ambilightEnabled && linkData.hoverZoomCaption) {
-                    hzCaption = $('<div/>', {id:'hzCaption', text:linkData.hoverZoomCaption}).css(hzCaptionCss).appendTo(hz.hzImg);
+                if (!options.ambilightEnabled && linkData.hoverZoomCaption) {
+                    if(options.captionLocation === "below") {
```
Comment (extesy) [inline js/hoverzoom.js]: please add a space between `if` and `(`.

File: js/hoverzoom.js [hunk 4]
```diff
@@ -593,8 +593,12 @@ var hoverZoom = {
...
+                        hzCaption = $('<div/>', {id:'hzCaption', text:linkData.hoverZoomCaption}).css(hzCaptionCss).appendTo(hz.hzImg);
+                    } else if(options.captionLocation === "above") {
```
Comment (extesy) [inline js/hoverzoom.js]: Same here.

**Comment by smaltby** [comment]
Changes made.

**Pull request merged**